lib/linter.js

```diff
@@ -382,7 +382,7 @@ class Linter {
       verbose: false,
     };
 
-    let PrettyPrinter = require('./printers/pretty');
+    let PrettyPrinter = require('./formatters/pretty');
```

Comment:
We should consider if we still support ember-cli-template-lint as of 3.0.0. I actually think we don't (because ember-cli-template-lint cannot pass a filePath, and we stopped supporting passing a moduleId/moduleName).

If that is the case, then we can probably remove this static method (since it was only supporting ember-cli-template-lint). 

That should be a separate PR though, just thought of this while reviewing.